index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portofolio</title>
+</head>
+<body>
+  <div class="container">
+    <div class="toolbar">
+      <div class="navbar">
+        <span class="navbar_welcome">Welcome</span>
+        <div class="navbar_menu">
+          <div class="navbar_menu_button">
```

Comment:
- [ ] I think it would be better if you wrap your hamburger in a` nav` tag instead of a div as the hamburger is part of navigation that you will be adding in your subsequent project. using a `nav` tag will make it easier for search engines and screen readers to understand that the section is for navigation thereby making your website really semantic 😊 

checkout this [ link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) for more info